spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+
+  describe "GET #show" do
+
+    before :each do
+      @post = FactoryGirl.create(:post)
+      @old_slug = @post.slug 
+      @post.update(title: 'Darkhorse post')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.